.github/workflows/diff_protobin.yml

```diff
@@ -62,12 +62,29 @@ jobs:
           const issue_number = context.issue.number;
           const githubToken = process.env.GITHUB_TOKEN;
 
+          // Get the existing comments.
+          const {data: comments} = await github.rest.pulls.listReviewComments({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: issue_number,
+          })
```

Comment:
While semicolons are not strictly required, and even though we dont have js linting in this project, and even if we did it probably would not parse the js inside a yaml workflow file, I would like to still add a semicolon here.